docs/unified-test-documentation/dasharo-stability/capsule-update.md

```diff
@@ -0,0 +1,340 @@
+1. Perform a capsule update using an invalid certificate file
+1. Reboot the system and select the UEFI Shell boot option.
+1. After the system boots, verify the BIOS version and save it as
+   `${updated_bios_version}`.
+1. Check that the BIOS version has not changed by verifying that
+   `${original_bios_version}` is equal to `${updated_bios_version}`.
+1. Verify that there is a status note about a security violation.
+
+**Expected result**
+
+1. The BIOS version should not change after attempting the capsule update.
+1. The status should contain a note indicating that the update was rejected due
+   to the invalid certificate.
+
+---
+
+## CUP002.001 Capsule Update With Wrong GUID (UEFI Shell)
+
+**Test description**
+
+This test aims to verify that the Device Under Test (DUT) rejects flashing a
+capsule with an invalid GUID.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Obtain and save the BIOS version as `${original_bios_version}`.
+1. Perform a capsule update using an invalid GUID file.
+1. Reboot the system and select the UEFI Shell boot option.
+1. After the system boots, verify and save the BIOS version as
+   `${updated_bios_version}`.
+1. Check that the BIOS version has not changed by verifying that
+   `${original_bios_version}` is equal to `${updated_bios_version}`.
+1. Verify that the status contains a `Not Ready` indication.
+
+**Expected result**
+
+1. The BIOS version should not change after attempting the capsule update.
+1. The status should contain a note indicating that the update was rejected due
+   to the invalid GUID.
+
+---
+
+## CUP130.001 Verifying BIOS Settings Persistence After Update - PART 1
+
+**Test description**
+
+This test aims to verify that BIOS settings persist after a Capsule Update.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Enter the Setup Menu.
+1. Navigate to the "Dasharo System Features" menu.
+1. Enter the "Networking Options" submenu.
+1. Enable the "Network Boot" option.
+1. Save the settings and reboot the system.
+1. Boot into UEFI Shell.
+1. Perform a capsule update using a valid capsule.
```

Comment:
This isn't part of this "test". This is only a preparatory step before next test case, to have a value to compare against after the update. Please remove those points.